engine/table/src/main/java/io/deephaven/engine/table/impl/util/InternalTableLoggerWrapper.java

```diff
@@ -0,0 +1,18 @@
+package io.deephaven.engine.table.impl.util;
+
+import io.deephaven.engine.table.TableDefinition;
+import io.deephaven.engine.table.impl.QueryTable;
+import io.deephaven.engine.tablelogger.UpdatePerformanceLogLogger;
+import io.deephaven.io.logger.Logger;
+import io.deephaven.tablelogger.TableLogger;
+
+public interface InternalTableLoggerWrapper<T extends TableLogger> {
```

Comment:
I would get rid of this class.  Logging the table and getting the result are two completely separate things, which may not be particularly related.  You should have provider for the TableLogger and another provider for the QueryTable.  In Community, those might need to be related; but in Enterprise they would not be.

Additionally, the idea of just "get the query table" doesn't make sense in Enterprise, because you should not want to monitor the performance from the executing query.